care.js

```diff
@@ -102,6 +101,62 @@ function tick() {
   doTheCodes();
 }
 
+function buildBoxes(showCommitsGraph) {
```

Comment:
because the `buildBoxes` manages multibe boxes the funtion parameter should not be that specific.
I would recommend removing the parameter and just use the global `config` internally.